test/unittests/qtMock/GraphPanelTest.cpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by stefan on 24.07.2023.
+//
+
+#include <gtest/gtest.h>
+#include <visualizer/graphPanel/GraphPanel.hpp>
+
+namespace {
+using age::visualizer::GraphPanel;
+}
+
+TEST(GraphPanelTest, construction) { GraphPanel g; }
+
+struct TestGraphPanel : GraphPanel {
+  void mousePressEvent(QMouseEvent* pEvent) override { this->GraphPanel::mousePressEvent(pEvent); };
```

Comment:
Remove this empty statement
```suggestion
  void mousePressEvent(QMouseEvent* pEvent) override { this->GraphPanel::mousePressEvent(pEvent); }
```